### PR TITLE
Updated the ca bundle for the SecretStore with new IBM Inter public c…

### PR DESCRIPTION
**Pull request by jadecarino**
…ertificate

- The caBundle value is the encoded IBM Root public certificate and the IBM Inter public certificate (okay to be checked in). IBM Inter needed updating.

### CODE REVIEW

File: infrastructure/cicsk8s/galasa-build/secret-manager-operator/secretStore.yaml [hunk 1]
```diff
@@ -13,7 +13,7 @@ spec:
...
...RUNoTXJTVzUwWlhKdVlYUnBiMjVoYkNCQ2RYTnAKYm1WemN5Qk5ZV05vYVc1bGN5QkRiM0p3YjNKaGRHbHZiakVkTUJzR0ExVUVBeE1VU1VKTklFbHVkR1Z5Ym1GcwpJRkp2YjNRZ1EwRXhEVEFMQmdOVkJBTVRCRU5TVERBd1FxQkFvRDZHUEdoMGRIQTZMeTlrWVhsdGRuTXhMbkJ2CmF5NXBZbTB1WTI5dE9qSXdNREV2VUV0SlUyVnlkaTlqWVdObGNuUnpVbTl2ZEM5RFVrd3dMbU55YkRBZEJnTlYKSFE0RUZnUVVCaS9heDVvZlA0ZFJ1VTNROXRxdWwwWktuY2d3SHdZRFZSMGpCQmd3Rm9BVStkNFk1WjR3RTJsUgpwLzE1aFVpTWZBNXYyT013Z1l3R0ExVWRJQVNCaERDQmdUQi9CZ01xQXdRd2VEQjJCZ2dyQmdFRkJRY0NBUlpxCmFIUjBjRG92TDNjekxUQXpMbWxpYlM1amIyMHZkSEpoYm5ObWIzSnRMM05oY3k5aGN5MTNaV0l1Ym5ObUwwTnYKYm5SbGJuUkViMk56UW5sVWFYUnNaUzlKYm1admNtMWhkR2x2Yml0VVpXTm9ibTlzYjJkNUsxTmxZM1Z5YVhSNQpLMU4wWVc1a1lYSmtjekFOQmdrcWhraUc5dzBCQVFzRkFBT0NBUUVBV3dJOG9wZzBIMXVrYlRMc2YrWFhSckhZCkJEVUIzcHdBQXBTaDlmcnVHYzNOdDVJYktEODY4N29pdXR4QkZ3SjB4eFBkUDEvQkwzSHljQ3F6U0RRRnRTVmQKVUFTSk9GMStHQjlETkR3cnJlT1dhVVRnMmNIQ0l4T2lqOUU5eXBVNjR3ZTlPTDI3Rmpjd1lLbkRicWZPcnN4cgp0clJheExtTTIrbE5hQkhJTXNBSWJNSDl2TjExRzdTcHpzWjVOcjVoWlhJbHhmcTkvSGZRdWpFNmhvQ2d0MmhvCk0waTRnUnlMZFZPYU5DSFBlb1ZPNm43T3NPS20zNkJnWFQ2MkIzNmU1cnZkS3p6YlVaWkxzZTlRUHR2UnljaHkKQVBBWDg5d2lISXFpRnN0ZXNXTCtrSW1wMUM5TDZpVk00REdid0dDcUt4VG5YMmFzaWthMHhIM2dXdW9aakE9PQotLS0tLUVORCBDRVJUSUZJQ0FURS0tLS0t
```
Comment (techcobweb) [inline infrastructure/cicsk8s/galasa-build/secret-manager-operator/secretStore.yaml:16]: What is this big thing ? As it's a yaml file, can you state it in a comment pls ?
ie: Is it a private or public certificate, or an API key, checked by what ? used for what ?

**Comment by galasa-team** [comment]
Build successful

File: infrastructure/cicsk8s/galasa-build/secret-manager-operator/secretStore.yaml (hunk 1 above)
Comment (jadecarino) [reply infrastructure/cicsk8s/galasa-build/secret-manager-operator/secretStore.yaml:19]: Comment added.

**Comment by galasa-team** [comment]
Build successful

**Review by techcobweb — APPROVED** [review]

**Pull request merged**